packages/prediction-app/src/components/sidebar/sidenav/Sidenav.module.css

```diff
@@ -0,0 +1,193 @@
+/* Most of this is copied from https://github.com/dhis2/design-specs/blob/master/src/components/sidenav.css */
+
+/* component-specific variables not defined in ui */
+html {
+    /* --sidenav-dark-bg: #040506;
+    --sidenav-dark-bg-hover: #0b0d10;
+    --sidenav-dark-bg-selected: #010101; */
+    --sidenav-dark-bg: #151B23;
+    --sidenav-dark-bg-hover: color-mix(
+        in srgb,
+        var(--sidenav-dark-bg),
+        white 10%
+    );
+    --sidenav-dark-bg-selected: color-mix(
+        in srgb,
+        var(--sidenav-dark-bg),
+        white 5%
+    );
+}
+
+/* nuke list styles */
+.sidenav-wrap ul {
+    list-style: none;
+    margin: 0;
+    padding-inline: 0;
+
+}
+
+/* wrapper */
+.sidenav-wrap {
+    width: 100%;
+    height: 100%;
+    background: var(--sidenav-dark-bg);
+    overflow-y: auto;
+    color: var(--colors-grey300);
+    display: flex;
+    flex-direction: column;
+}
+
+.sidenav-items {
+    overflow-y: auto;
+    scrollbar-color: var(--colors-grey700) var(--sidenav-dark-bg);
+    padding-block-start: var(--spacers-dp8);
+    scrollbar-width: thin;
+}
+
+/* parent */
+.sidenav-parent button {
+    border: none;
+    background: var(--sidenav-dark-bg);
+    color: var(--colors-grey300);
+    font-size: 16px;
+    text-align: left;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    min-height: 32px;
+    padding: 8px 8px 8px 12px;
+}
+.sidenav-parent button:hover {
+    text-decoration: underline;
```

Comment:
What did we decide on when it comes to the underline? Personally, I don't love it, but I'm fine if you do.

Reply:
I think I just readded it for hover/focus? I muted the strength of the hover color to be more inline with the original design, and thus felt it was needed to have some more feedback for those active styles.